### ISSUE TITLE
Addin discoverer failing

### ISSUE THREAD

**Issue by pascalberger** [cake-contrib/Cake.AddinDiscoverer#118]
Addin discoverer currently seems to be failling: 

```
***** AN EXCEPTION HAS OCCURED *****
Octokit.NotFoundException: repos/cake-contrib/Cake.Recipe/contents/tools/packages.config was not found.
   at async Task<IReadOnlyList<T>> Octokit.ApiPagination.GetAllPages<T>(Func<Task<IReadOnlyPagedCollection<T>>> getFirstPage, Uri uri)
   at async Task<SemVersion> Cake.AddinDiscoverer.Steps.UpdateCakeRecipeStep.FindCakeVersionUsedByRecipe(DiscoveryContext context)
   at async Task Cake.AddinDiscoverer.Steps.UpdateCakeRecipeStep.ExecuteAsync(DiscoveryContext context)
   at async Task<ResultCode> Cake.AddinDiscoverer.AddinDiscoverer.LaunchDiscoveryAsync()
Cake.exe : An error occurred when executing task 'Run'.
At line:1 char:1
```

https://ci.appveyor.com/project/cakecontrib/cake-addindiscoverer/builds/35287831

**Comment by Jericho** [comment]
Yes I am aware. This affecting the ability to update Cake.Recipe but the markdown and Excel reports are Thankfully not affected by this problem. 

This is due to the fact that AddinDisco Relies on the presence of tools\package.config in the Cake.Repo but this file was removed earlier this week and a different file is now used to pin the version of Cake. I need to adjust the AddinDisco to rely on this alternate file.

**Comment by Jericho** [comment]
Fixed in 3.48.0
